src/locales/pt/translation.json

```diff
@@ -109,6 +109,8 @@
     "downloads_path": "Diretório dos downloads",
     "change": "Mudar",
     "notifications": "Notificações",
+    "menu_options": "Opções",
+    "Shutdown_After_Downloads": "Desligar após o término do download",
```

Comment:
Please use snake_case `shutdown_after_downloads`
Plus, add translation for all the other supported languages (fr, es, en)
You can use i18n.ally to do automatically